sql_intermediate/sql_intermediate.md

```diff
@@ -881,6 +590,7 @@ ORDER BY earliest_birthdate;
 </div>
 
 <div class = "warning">
+<b style="color: rgb(var(--color-highlight));">Warning!</b><br>
```

Comment:
```suggestion
<div class = "help">
<b style="color: rgb(var(--color-highlight));">Troubleshooting help</b><br>
```

This feels like more of a help box than warning to me, but feel free to push back